crates/runtime/src/embeddings/vector_search.rs

```diff
@@ -174,9 +175,30 @@ impl VectorSearch {
 
                     let outtt: Vec<String> =
                         outt.iter().flat_map(std::clone::Clone::clone).collect();
+                    /// Retrieve the column data for just the primary keys
+                    /// This could be empty or also include the embedding column.
+                    let public_key_data = batch
```

Comment:
Why is this called `public_key_data`? Shouldn't it be called `primary_key_data`?